tests/Test/Xrefcheck/TrailingSlashSpec.hs

```diff
@@ -45,6 +45,4 @@ spec = do
       [ "tests/markdowns/without-annotations"
       , "tests/markdowns/without-annotations/"
       , "tests/markdowns/without-annotations/./"
-      , "tests/markdowns/without-annotations/all_checked.md"
-      , "tests/markdowns/without-annotations/./all_checked.md"
```

Comment:
Why were these tests removed?

Reply:
These paths seem to be passed as `root`s while gathering repo info. Since now we forbid verifying a single file as `root`, these paths fail the test

Reply:
Ah right, makes sense!